src/components/Tokens/TokenDetails/BalanceSummary.tsx

```diff
@@ -61,49 +69,213 @@ const StyledNetworkLabel = styled.div`
   line-height: 16px;
 `
 
-export default function BalanceSummary({ token }: { token: Currency }) {
-  const { account, chainId } = useWeb3React()
-  const theme = useTheme()
-  const { label, color } = getChainInfo(asSupportedChain(chainId) ?? ChainId.MAINNET)
-  const balance = useCurrencyBalance(account, token)
-  const { formatCurrencyAmount } = useFormatter()
+interface BalanceProps {
+  token?: Currency
+  chainId: ChainId
+  balance?: CurrencyAmount<Currency> // TODO(WEB-3026): only used for pre-Info-project calculations, should remove after project goes live
```

Comment:
left comment bc it is not immediately clear & may be missed on feature-flag code cleanup